### DISCUSSION TITLE
TraderPlus?

### DISCUSSION THREAD

**Discussion by djsiner-bps** [Gh0st352/DNA-Keycards-eEditor#8] [Q&A] [answered]
Any way to get this to work with TraderPlus?

**Comment by Gh0st352** [comment, marked as answer]
> Any way to get this to work with TraderPlus?

Sorry, I never received a notification for a new discussion post; just seeing this now!

I don't have plans to include automatic import for TraderPlus lists (I don't use that personally), BUT you can manually copy paste typename lists into the desired fields rather than using the Expansion Market Import.

Just ensure 1 typename per line, no spaces/whitespaces, no empty lines before you generate :)

**Comment by djsiner-bps** [comment]
Awesome! I will def Try that out. Thank you!

**Comment by CAG101** [comment]
When I try to add the types file for a couple of different clothing mods the app crashes.  Is this a known bug with a remedy?

**Reply by Gh0st352** [reply]
See https://github.com/Gh0st352/DNA-Keycards-eEditor/issues/9

I've had a report or two about this in the past for various mods, but no one provides me files to troubleshoot it. (I can't reproduce and fix on my end without the files people have an issue with). 

If you want me to investigate, please create a new issue and provide all files used/steps to reproduce :)
